languages/french.js

```diff
@@ -201,22 +201,22 @@ module.exports = {
         field: (prefix, member) => `Écrivez \`${prefix}invites ${member.user.tag}\` pour voir le nouveau nombre d'invitations de **${member.user.username}** !`
     },
 
-    blacklist: {
+  blacklist: {
```

Comment:
```suggestion
    blacklist: {
```
Two spaces are missing here